cms/models.py

```diff
@@ -90,4 +90,25 @@ class CoursePage(ProductPage):
         on_delete=models.SET_NULL,
         help_text="The course for this page",
     )
-    content_panels = [FieldPanel("course")] + ProductPage.content_panels
+    video_url = models.URLField(
+        null=True,
+        blank=True,
+        help_text="URL to the video to be displayed for this course",
+    )
+    video_title = RichTextField(
+        blank=True, help_text="The title to be displayed for the course video"
+    )
+    background_image = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Background image size must be at least 1920x1080 pixels.",
+    )
```

Comment:
I'm pretty sure all of these need to be defined for `ProgramPage` as well. Have Peter or Ferdi told you that the program detail pages will not have these fields? Are we just limiting this to course pages for simplicity right now?

Reply:
I've moved this to `ProductPage`.

Reply:
I've also moved common properties to `PageProperties` mixin in the Django model where a lot of code was being duplicated.